### PR TITLE
Pop up-Project

### PR DESCRIPTION
**Pull request by amanuelbelete95**
**List of Completed Activities**

- When the buttons are clicked, the popup with details about the project appears.
- The popup disappears when the user clicks (or taps) the close (X) button.
- Used JavaScript array to store all of the information for all projects.
- implemented popups for both mobile and desktop screen sizes
- Used Figma disgn as a reference while implementing the popup window

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -18,3 +18,100 @@ function menuRemover() {
...
+   </div> 
+  </div>
+    </div>
+`;
+
+const openBtn = document.querySelectorAll('[data-modal-target]');
+const closeModalButtons = document.querySelectorAll('[data-close-button]');
+const overlay = document.querySelector('.overlay');
+
+function openModal(modal) {
```
Comment (Olamarx) [inline script.js]: - [ ]  I would recommend you use the ES6 arrow function. This is for best practice and shows you are up to date in the industry.

File: script.js [hunk 2]
```diff
@@ -18,3 +18,100 @@ function menuRemover() {
...
+  if (modal == null) return;
+  modal.classList.toggle('active');
+  overlay.classList.toggle('active');
+}
+
+function closeModal(modal) {
```
Comment (Olamarx) [inline script.js]: - [ ]  I would recommend you use the ES6 arrow function. This is for best practice and shows you are up to date in the industry.

File: script.js [hunk 3]
```diff
@@ -18,3 +18,100 @@ function menuRemover() {
...
+
+   <ul class="pop-list">
+    <li>${popWindow[0].popList[0]}</li>
+    <li>${popWindow[0].popList[1]}</li>
+    <li>${popWindow[0].popList[2]}</li>
+   </ul>
+
+   <div class="image-info-container">
+    <div class="img-pop">
+    <img src="${popWindow[0].popImg}"/>
```
Comment (Olamarx) [inline script.js:67]: - [ ] This image is broken, kindly fix it

image

File: script.js [hunk 4]
```diff
@@ -18,3 +18,100 @@ function menuRemover() {
...
+   </div>
+   
+   <div class="info-button">
+    <p ${popWindow[0].popDescription}class="pop-desktop-info">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent</p>
+    <div class="my-buttons">
+    <button class="btn1">See live <span></span><img src="${popWindow[0].poplive}"></button>
+    <button class="btn1">See source <span></span><img src="${popWindow[0].popGitHub}"></button>
```
Comment (Olamarx) [inline script.js:74]: - [ ] These images are broken, kindly fix it

image

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @amanuelbelete95,

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linters passed ✔️
- popup functionality ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Whoistolu — COMMENTED** [review]
Hi @amanuelbelete95 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### You have submitted a project, that has a failed linter test. Kindly ensure all linters pass before requesting another review. Linters help us check for code standards and it is also a hard requirement.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: script.js [hunk 5]
```diff
@@ -18,3 +18,112 @@ function menuRemover() {
...
+  if (modal == null) return;
+  modal.classList.toggle('active');
+  overlay.classList.toggle('active');
+};
+
+// function closeModal(modal) {
+//   if (modal == null) return;
+//   modal.classList.remove('active');
+//   overlay.classList.remove('active');
+// }
```
Comment (leonardodiasb) [inline script.js]: - [ ] Kindly remove commented-out code. They can make your project looks messy and they don't add any functionality. For better readability and clean code, please remove them.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @amanuelbelete95, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - No linter errors. ✔️
 - Professional README. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AnselemOdims — APPROVED** [review]
### Hi  @amanuelbelete95 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**